ui/src/app/subnets/urls.ts

```diff
@@ -7,7 +10,8 @@ import type { VLAN, VLANMeta } from "app/store/vlan/types";
 import { argPath, isId } from "app/utils";
 
 const urls = {
-  index: "/networks", // ?by = 'fabric' | 'space'
+  index: ({ by }: { by?: GroupByKey } = {}): string =>
```

Comment:
I think maybe this should be left as `index` and then add another url, something like `indexBy` with the querystring. That way we can add a redirect from index to indexBy to add the querystring if it doesn't already exist.